test/e2e/data/cni/kindnet/kindnet.yaml

```diff
@@ -77,7 +77,7 @@ spec:
                 fieldRef:
                   fieldPath: status.podIP
             - name: POD_SUBNET
-              value: '${DOCKER_POD_CIDRS}'
+              value: '${DOCKER_POD_CIDRS},${DOCKER_POD_IPV6_CIDRS}'
```

Comment:
Is the order relevant here?

Reply:
Not relevant AFAIK - these CIDRs are handled for ensuring pods/services are routed correctly. When dualstack both loops - IPv4 and IPv6 - in Kindnet run concurrently.